DKBeatMaker/DKBeatMaker/ArrangementView/ArrangementViewPresenterDelegate.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+protocol ArrangementViewPresenterDelegate {
```

Comment:
I usually name the delegate as a ViewSurface. The term ViewSurface indicates to me that any class conforming to this is a ViewController. Not that you would ever cast it, but it is good to indicate that this protocol is used for delegation between a presenter and a view.